nirc_ehr/src/org/labkey/nirc_ehr/columnTransform/NIRCUserCreateTransform.java

```diff
@@ -0,0 +1,188 @@
+package org.labkey.nirc_ehr.columnTransform;
+
+import org.labkey.api.data.SQLFragment;
+import org.labkey.api.data.SqlSelector;
+import org.labkey.api.di.columnTransform.ColumnTransform;
+import org.labkey.api.di.columnTransform.ColumnTransformException;
+import org.labkey.api.query.QueryService;
+import org.labkey.api.query.UserSchema;
+import org.labkey.api.security.Group;
+import org.labkey.api.security.InvalidGroupMembershipException;
+import org.labkey.api.security.MemberType;
+import org.labkey.api.security.SecurityManager;
+import org.labkey.api.security.User;
+import org.labkey.api.security.UserManager;
+import org.labkey.api.security.ValidEmail;
+import org.labkey.nirc_ehr.query.NIRC_EHRUserSchema;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class NIRCUserCreateTransform extends ColumnTransform
+{
+    private transient Map<String, Integer> _sourceUserNameMap = new HashMap<>();
+
+    /**
+     * Search for the user by the email prefix (ie. jdoe@labkey.com matches jdoe).
```

Comment:
Comment doesn't match the implementation - it's using name, not email username

Reply:
Good catch, fixed.